layouts/partials/tw/list-callout-story.html

```diff
@@ -22,7 +24,7 @@
     </h2>
   {{ end }}
 
-  {{ range $page | first 1 }}
+  {{ range $pages | first 1 | complement $featuredStory }}
```

Comment:
If it's in this order, it might end up empty. I would do `$pages | complement $featuredStory | after 1`.